book/src/run_a_node.md

```diff
@@ -26,6 +26,7 @@ per beacon node. The reason for this is that the beacon node *controls* the exec
 - [Besu](https://besu.hyperledger.org/en/stable/public-networks/get-started/connect/mainnet/)
 - [Erigon](https://github.com/ledgerwatch/erigon#beacon-chain-consensus-layer)
 - [Geth](https://geth.ethereum.org/docs/getting-started/consensus-clients)
+- [Reth](https://reth.rs/run/run-a-node.html)
```

Comment:
I think for the general users looking to setup a node, we can link directly to the page of running a mainnet/testnet Reth node here.

```suggestion
- [Reth](https://reth.rs/run/mainnet.html)
```